src/storage/sftp/storage.c

```diff
@@ -877,6 +878,102 @@ storageSftpNew(
             }
         }
 
+        // Check for a matching entry in the .ssh/known_hosts file
+        if (param.requireKnownHostsMatch)
+        {
+            LIBSSH2_KNOWNHOSTS *nh;
```

Comment:
"nh" seems like a pretty terrible variable name for this...

Reply:
:slightly_frowning_face: you are correct, that was a straight carryover from the example code w/o thinking on my part. Will replace with a descriptive name.